howto-promote.dd

```diff
@@ -83,7 +85,6 @@ $(H3 Promoting the Project)
 	)
 
 	$(UL
-	$(LI $(LINK2 http://www.ddj.com, Dr. Dobb's Journal))
```

Comment:
afaict it's archive-only:
http://www.drdobbs.com/architecture-and-design/farewell-dr-dobbs/240169421